src/test/compile-fail/self_type_keyword.rs

```diff
@@ -15,19 +15,24 @@ struct Self;
 
 struct Bar<'Self>;
 //~^ ERROR lifetimes cannot use keyword names
+//~^^ ERROR parameter `'Self` is never used
```

Comment:
Could we move the `struct Self;` in this test into a sub-module to avoid these extra errors?